zeppelin-web/src/app/jobmanager/jobmanager.controller.js

```diff
@@ -0,0 +1,211 @@
+/*jshint loopfunc: true, unused:false */
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+angular.module('zeppelinWebApp')
+  .filter('myJob', function() {
+
+    function filterContext(jobItems, filterConfig) {
+      var FILTER_VALUE_INTERPRETER = filterConfig.FILTER_VALUE_INTERPRETER;
+      var FILTER_VALUE_NOTEBOOK_NAME = filterConfig.FILTER_VALUE_NOTEBOOK_NAME;
```

Comment:
Why var names are in upper case? Is it consistent with the style of other frontend code we have?